sdk/src/main/java/io/outbound/sdk/Outbound.java

```diff
@@ -48,7 +45,7 @@ static void initForTesting(Application app, String apiKey, String gcmSenderId,
      * If there is no active user, then the value returned will be null.
      */
     public static String getActiveToken() {
-        return OutboundClient.getInstance().fetchCurrentGCMToken();
+        return OutboundClient.getInstance().fetchCurrentFCMToken();
```

Comment:
Tiny nitpick: the convention in Java is to camel-case the acronyms too, so technically this should be `fetchCurrentFcmToken` :D

Reply:
Whoops I hadn't even thought of that, I was just replacing 'G' with 'F' wherever I saw it